File: Course-Project/app/src/main/java/com/vladislavmyasnikov/courseproject/data/models/Task.java
package com.vladislavmyasnikov.courseproject.data.models;

import com.google.gson.annotations.SerializedName;

public class Task {

    @SerializedName("id")
    public int id;

    @SerializedName("title")
    public String title;

    @SerializedName("max_score")
    public double maxScore;

    @SerializedName("deadline_date")
    public String deadlineDate;
}

File: Course-Project/app/src/main/java/com/vladislavmyasnikov/courseproject/data/models/Task.kt
package com.vladislavmyasnikov.courseproject.data.models

import com.google.gson.annotations.SerializedName

class Task {

    @SerializedName("id")
    var id: Int = 0

    @SerializedName("title")
    var title: String? = null

    @SerializedName("max_score")
    var maxScore: Double = 0.toDouble()

    @SerializedName("deadline_date")
    var deadlineDate: String? = null
}
